design-doc-template.adoc

```diff
@@ -97,12 +97,22 @@ categories:
 
 //== Implementation Plan
 ////
-Delete if not needed. The intent is if you have a complex feature which can 
-not be delivered all in one go to suggest the strategy. If your feature falls 
-into this category, please mention the Release Coordinators on the pull 
+Delete if not needed. The intent is if you have a complex feature which can
+not be delivered all in one go to suggest the strategy. If your feature falls
+into this category, please mention the Release Coordinators on the pull
 request so they are aware.
 ////
 
+== Admin Clients
+
+////
+Identify the level of compatibility this feature will have with the existing
+admin clients (JBoss CLI and the Admin Console / HAL).
+
+Identify any follow up work that will be required in the clients and link issues
+created to track this work.
+////
```

Comment:
Please use the markdown style illustrated in https://github.com/wildfly/wildfly-proposals/pull/581/files#diff-1e46fd5b24b8df67bec2629d044ee017dc0e62542e582563b1abf9a29db7dfe6.

Reply:
Updated